.github/workflows/snyk.yml

```diff
@@ -0,0 +1,15 @@
+name: Snyk Monitor Scanning
+on:
+ release:
+   types: [published] 
+ push:
+    branches:
+      - 'main' 
+      - 'master'
```

Comment:
we don't really have / use `master` branch

Reply:
Understood, this is a standard workflow file we're using in a few repos. Some still use the the old default branch naming, I can remove it.